source/dub/package_.d

```diff
@@ -169,8 +171,11 @@ class Package {
 			version_override = Optional version to associate to the package
 				instead of the one declared in the package recipe, or the one
 				determined by invoking the VCS (GIT currently).
+			mode = The `StrictMode` to use.
```

Comment:
The backtip creates a link, doesn't it ? But yeah I'll expand on that

Reply:
backticks just seem to make it a code block (vanilla dmd ddoc and also adrdox)

`REF` and `LREF` seem to be macros defined in phobos/druntime, but they are also understood by adrdox without any defined macros, so I think it's best to use them.

Reply:
Eh the documentation processor should really automatically create links to types within code blocks, instead of doing it with random names and forcing us to escape things :facepalm: